node.gyp

```diff
@@ -718,14 +647,33 @@
             }]
           ],
           'action': [
-            'python',
-            'tools/js2c.py',
+            'python', 'tools/js2c.py',
             '<@(_outputs)',
-            '<@(_inputs)',
+            '<@(_inputs)', 'config.gypi',
           ],
         },
       ],
-    }, # end node_js2c
+    }, # node_lib_target_name
+    {     # generate ETW header and resource files
```

Comment:
Tiny nit: This comment might look better in the previous line.

Reply:
Moving it was a cosmetic desittion:
![image](https://user-images.githubusercontent.com/96947/46353010-24e8de80-c629-11e8-931f-87541ac805c4.png)
I could move it one line lower to the top of the target